### PR TITLE
comment/hint for bibliography

### PR DESCRIPTION
**Pull request by phip1611**
Small hint. This is something everyone has to decide by themselves. Therefore, I've only given the pointer to this option in the comment.

### CODE REVIEW

**Comment by Nils-TUD** [comment]
Good hint! I think we should go one step further for the author list, though. It can be acceptable to not list all authors if there is a page limit for references. However, such limit does not exist for a student thesis. Therefore, listing all authors should be the default to give credit to all authors (I am actually wondering why it is not already the default).

**Comment by phip1611** [comment]
I've updated the PR @Nils-TUD

File: preamble/packages.tex [hunk 1]
```diff
@@ -1,4 +1,6 @@
-\usepackage[backend=biber,style=alphabetic,alldates=long]{biblatex}
+% You can chose style "numeric" instead which is common in many papers.
```
Comment (blitz) [inline preamble/packages.tex:1]: nit: "choose"

File: preamble/packages.tex [hunk 2]
```diff
@@ -1,4 +1,6 @@
...
+% Without "maxbibnames=99" the biblopgraphy entries only contain "First Name et al."
```
Comment (blitz) [inline preamble/packages.tex:2]: nit: bibliography?

**Review by blitz — CHANGES_REQUESTED** [review]
Looks good when the typos are fixed.

**Comment by phip1611** [comment]
Fixed @blitz

**Pull request merged**